File: 신명준/평균.cpp
#include<stdio.h>
double  arr[10001];

int main()
{
    int size;
    scanf("%d",&size);
    double  max =0;
    for(int i =0; i<size ;i++)
    {
        scanf("%lf",&arr[i]);
        if(arr[i] >max)
            max =arr[i];
    }
    
    double sum =0;
    for(int i =0; i<size ;i++)
    {
       sum += (arr[i]/max)  *100;
    }
    printf("%lf\n",sum/size);
    
}

File: 정다은/개인문제풀이/0530 스터디/1546. 평균/평균.java
import java.util.Scanner;

public class Main {
	
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		int[] arr = new int[n];
		int max = 0;
		int sum = 0;
		
		for(int i = 0; i < n; i++) {
			arr[i] = sc.nextInt();
			max = Math.max(max, arr[i]);
			sum += arr[i];
		}
			
		System.out.println((double)sum / max * 100 / n);
		sc.close();
	}
	
	

}
